aws/http_client_test.go

```diff
@@ -29,33 +27,3 @@ func TestBuildableHTTPClient_NoFollowRedirect(t *testing.T) {
 		t.Errorf("expect %v code, got %v", e, a)
```

Comment:
Should this be fatal too? Since it would be a network error.

Reply:
I think in this case since the assertion failing wouldn't prevent further assertions being performed the `Errorf` is good. `Fatalf` is great further assertions may not be testable.